src/OpenColorIO/CMakeLists.txt

```diff
@@ -0,0 +1,108 @@
+set(SOURCES
+	Baker.cpp
+	BitDepthUtils.cpp
+	Caching.cpp
+	ColorSpace.cpp
+	Config.cpp
+	Context.cpp
+	Display.cpp
+	Exception.cpp
+	fileformats/FileFormat3DL.cpp
+	fileformats/FileFormatCCC.cpp
+	fileformats/FileFormatCC.cpp
+	fileformats/FileFormatCDL.cpp
+	fileformats/FileFormatCSP.cpp
+	fileformats/FileFormatDiscreet1DL.cpp
+	fileformats/FileFormatHDL.cpp
+	fileformats/FileFormatICC.cpp
+	fileformats/FileFormatIridasCube.cpp
+	fileformats/FileFormatIridasItx.cpp
+	fileformats/FileFormatIridasLook.cpp
+	fileformats/FileFormatPandora.cpp
+	fileformats/FileFormatResolveCube.cpp
+	fileformats/FileFormatSpi1D.cpp
+	fileformats/FileFormatSpi3D.cpp
+	fileformats/FileFormatSpiMtx.cpp
+	fileformats/FileFormatTruelight.cpp
+	fileformats/FileFormatVF.cpp
+	GpuShader.cpp
+	GpuShaderDesc.cpp
+	GpuShaderUtils.cpp
+	HashUtils.cpp
+	ImageDesc.cpp
+	ImagePacking.cpp
+	Logging.cpp
+	Look.cpp
+	LookParse.cpp
+	MathUtils.cpp
+	md5/md5.cpp
+	OCIOYaml.cpp
+	Op.cpp
+	OpOptimizers.cpp
+	ops/Allocation/AllocationOp.cpp
+	ops/Exponent/ExponentOps.cpp
+	ops/Log/LogOps.cpp
+	ops/Lut1D/Lut1DOp.cpp
+	ops/Lut3D/Lut3DOp.cpp
+	ops/Matrix/MatrixOps.cpp
+	ops/NoOp/NoOps.cpp
+	ops/Truelight/TruelightOp.cpp
+	OpTools.cpp
+	ParseUtils.cpp
+	PathUtils.cpp
+	Platform.cpp
+	Processor.cpp
+	ScanlineHelper.cpp
+	Transform.cpp
+	transforms/AllocationTransform.cpp
+	transforms/CDLTransform.cpp
+	transforms/ColorSpaceTransform.cpp
+	transforms/DisplayTransform.cpp
+	transforms/ExponentTransform.cpp
+	transforms/FileTransform.cpp
+	transforms/GroupTransform.cpp
+	transforms/LogTransform.cpp
+	transforms/LookTransform.cpp
+	transforms/MatrixTransform.cpp
+	transforms/TruelightTransform.cpp
+	UnitTest.cpp
+)
+
+add_library(OpenColorIO ${SOURCES})
+set_property(TARGET OpenColorIO PROPERTY CXX_STANDARD 11)
```

Comment:
The C++ version selection is already done in the CMakeLists.txt line 5. Why is that needed here too?
If oneday the C++ version is changed, only one location should be changed.

Reply:
Correct, This was leftover from an earlier test. I've removed the additional check.